prow/oss/terraform/modules/alerts/main.tf

```diff
@@ -212,6 +212,7 @@ resource "google_monitoring_alert_policy" "probers" {
       fetch uptime_url
       | metric 'monitoring.googleapis.com/uptime_check/check_passed'
       | align next_older(1m)
+      | filter resource.project_id == '${var.project}'
```

Comment:
are we sure `project_id` is a reliable differentiator?

Reply:
Yes it is. Previously I was setting the value based on the `blackbox_prober` map values. Now they're all set to a consistent value (the metrics project).